ActivityLauncherApp/src/main/AndroidManifest.xml

```diff
@@ -43,10 +43,20 @@
         <activity
             android:name=".RootLauncherActivity"
             android:label="@string/context_action_launch_as_root"
-            android:exported="true" >
+            android:exported="true"
+            android:theme="@style/AppTheme.NoDisplay">
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".LauncherActivity"
+            android:exported="true"
+            android:theme="@style/AppTheme.NoDisplay">
+            <intent-filter>
+                <action android:name="activitylauncher.intent.action.LAUNCH_SHORTCUT"/>
```

Comment:
Why new activity? Does android 13 forbid crafting shortcuts die other apps, and this we use the root launcher approach for all shortcuts now?

Reply:
So, I don't really know why some activities do work and some don't. At first, I thought it was because the activity had $ in the name, but that is not the case. So basically no idea. Launch works fine and when using shortcuts it says the application doesn't exist. I have checked the intent we are setting with the debugger and nothing seems off.